python/graphscope/framework/utils.py

```diff
@@ -33,6 +34,44 @@
 from graphscope.proto import types_pb2
 
 
+def is_free_port(port, host="localhost", timeout=1.0):
```

Comment:
1 second? Please double check whether this empirical value is reasonable for a port testing.

Reply:
following [mysql](https://www.programcreek.com/python/?code=mysql%2Fmysql-utilities%2Fmysql-utilities-master%2Fmysql-test%2Fmutlib%2Fmutlib.py), change timeout to 0.2